db/migrate/20220402151141_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] I would recommend naming `author_id` as a column name instead of `user_id` as per [ER diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) and give reference to table users in the migration file.

Also, do the same with the posts and likes table.

I'm here to give hint to add as an `author_id` column in the comments table with reference to table users.
Hint: 
```ruby
create_table :comments do |t|
  ...
  t.references :author, foreign_key: { to_table: 'users' } # this will create author_id and a corresponding index
   
end
```